ffi/capi/src/calendar.rs

```diff
@@ -101,6 +101,7 @@ pub mod ffi {
         /// Returns the kind of this calendar
         #[diplomat::rust_link(icu::calendar::AnyCalendar::kind, FnInEnum)]
         #[diplomat::attr(auto, getter)]
+        #[diplomat::attr(demo_gen, disable)] // trivial
```

Comment:
Praise: Thanks for adding a comment on why you are disabling.

Please do this everywhere. I encourage more than 1 word.